deploy/codex/.env

```diff
@@ -41,6 +41,7 @@ WBIA_DB_DIR=/data/db
 HOUSTON_DB_NAME=houston
 HOUSTON_DB_USER=houston
 HOUSTON_DB_PASSWORD=development
+HOUSTON_URL=http://houston:5000/
```

Comment:
Wouldn't it be localhost:83 to hit the houston container? I think I'm missing something here.

Reply:
Not entirely sure but I think this is correct as it's analogous to ACM_AUTHENTICATIONS_URI__DEFAULT below